src/utils/map-controller.js

```diff
@@ -28,9 +28,9 @@ import type {MjolnirEvent} from 'mjolnir.js';
 const NO_TRANSITION_PROPS = {
   transitionDuration: 0
 };
-const LINEAR_TRANSITION_PROPS = {
+export const LINEAR_TRANSITION_PROPS = {
   transitionDuration: 300,
-  transitionEasing: t => t,
+  transitionEasing: (t: number) => t,
   transitionInterpolator: new LinearInterpolator(),
   transitionInterruption: TRANSITION_EVENTS.BREAK
```

Comment:
Will the `TransitionManager`'s default props need to match these, too? If so, is it worth importing these into `utils/transition-manager`?

Reply:
Technically no. The default props of `TransitionManager` is the default behavior when user only supply `transitionDuration`, which can be anything we want. The controller's `LINEAR_TRANSITION_PROPS` intend to match the behavior of Mapbox.